### ISSUE TITLE
Добавить свойство fixedProperty

### ISSUE THREAD

**Issue by IIIu6ko** [AddMoreScripts/hystModal#16]
Есть замечательное свойство fixedSelectors, оно отлично работает, но только для fixed элементов, которые не занимают 100% ширины окна.

Предлагаю добавить свойство fixedProperty, в которое можно будет передать строку с названием css свойства. Например, для элементов со 100% шириной margin-right не работает, элемент всё равно дёргается при открытии модалки, но зато отлично работает padding-right. В некоторых ситуациях ещё может пригодиться просто width.

Идею честно подглядел в [scroll-lock](https://github.com/FL3NKEY/scroll-lock/blob/master/README.RU.md#setfillgapmethodfillgapmethod) :)

**Comment by AddMoreScripts** [comment]
Да, спасибо за предложение, попробую продумать и добавить в будущем.
